### ISSUE TITLE
Crossfilter Bar Charts spillover elements

### ISSUE THREAD

**Issue by cheelee** [openworm/movement_cloud#87]
<img width="1158" alt="crossfilterchartbug" src="https://user-images.githubusercontent.com/316749/27081467-14d502f8-5073-11e7-943f-9c23708151c9.png">

For some charts, the right-most edge sometimes loses elements that cannot be properly selected by the sliders ...

**Comment by MichaelCurrie** [comment]
Yes I noticed this but didn't have time to fix it.  Probably an off-by-one error or something.  We should fix this at some point.

**Comment by cheelee** [comment]
I've implemented a workaround. It is not pretty, but now the sliders will extend to the the edge of the margins. For some reason the SVG graphics is asked to scale the bars to extend all the way out to the margins, but I cannot yet find the code that does this. I did find the code that tells the brush slider its extent, and this does the trick for now.

Until we get a proper fix in, this is an improvement over the previous implementation :)
